src/servers/src/http.rs

```diff
@@ -769,6 +761,7 @@ impl HttpServer {
     fn route_log<S>(log_state: LogState) -> Router<S> {
         Router::new()
             .route("/logs", routing::post(event::log_ingester))
+            .route("/loki/api/v1/push", routing::post(event::loki_ingest))
```

Comment:
Should we place the path under `events`, or move to the top level like Prometheus protocol?